vm-image-spec.yaml

```diff
@@ -233,6 +233,22 @@ files:
         values: [approximate_working_set_size]
         query: |
           select neon.approximate_working_set_size(false) as approximate_working_set_size;
+
+      - metric_name: current_lsn
+        type: gauge
+        help: 'Current LSN of the database'
+        key_labels:
+        values: [lsn]
+        query: |
+          select pg_current_wal_lsn() as lsn;
```

Comment:
`pg_current_wal_lsn' throws an error in a standby server. Is that OK?

In a standby, `pg_last_wal_receive_lsn` and `pg_last_wal_replay_lsn` would be interesting, instead.

Reply:
From testing, it doesn't seem to be an error, it just returns null. But I added a case to return `pg_last_wal_replay_lsn` if it's a standby